wbia/dtool/copy_sqlite_to_postgres.py

```diff
@@ -77,7 +77,7 @@ def get_random_rows(
     ):
         total_rows = self.get_total_rows(schema, table_name)
         rows_to_check = max(int(total_rows * percentage), min_rows)
-        if max_rows > 0:
+        if max_rows > 0 and not (max_rows <= -1):
```

Comment:
@mmulich `max_rows > 0` already includes `not (max_rows <= -1)` right?  Or is the condition wrong?  Max rows is only meant to apply if it's greater than 0.